### PR TITLE
content modification

### PR DESCRIPTION
**Pull request by Nithinvanamala**
this is my intro

### CODE REVIEW

File: ram.txt [hunk 1]
```diff
@@ -1,4 +1,9 @@
-helloo
-hiii
-iam ram
-hi seetha
+iam nithin 
+iam currently 
+persuing btech final year at 
+geethanjali college of engineering and technology
```
Comment (Abhi-1442) [inline ram.txt]: College name should be in Camel Case

File: ram.txt [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+cse
+java
+python 
+ai 
+ml
```
Comment (Abhi-1442) [inline ram.txt]: Should be in uppercase

**Review by Abhi-1442 — CHANGES_REQUESTED** [review]

File: ram.txt [hunk 3]
```diff
@@ -1,4 +1,9 @@
-helloo
-hiii
-iam ram
-hi seetha
+iam nithin
```
Comment (KONDAHARSHITA) [inline ram.txt]: Name should start with capital .

**Review by Nithinvanamala — COMMENTED** [review]
i thought everything in format

File: ram.txt (hunk 3 above)
Comment (Abhi-1442) [inline ram.txt]: Start with uppercase

**Review by Abhi-1442 — APPROVED** [review]

**Review by KONDAHARSHITA — APPROVED** [review]

File: ram.txt [hunk 4]
```diff
@@ -1,4 +1,8 @@
-helloo
-hiii
-iam ram
-hi seetha
+iam Nithin 
+iam currently 
+persuing btech final year at
```
Comment (Abhi-1442) [inline ram.txt]: Add this line to above line itself

**Review by Abhi-1442 — CHANGES_REQUESTED** [review]

**Review by KONDAHARSHITA — APPROVED** [review]

**Comment by Nithinvanamala** [comment]
problems are resolved

**Pull request merged**